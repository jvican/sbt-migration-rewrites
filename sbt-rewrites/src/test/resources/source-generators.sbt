lazy val seed = taskKey[File]("Seed.")
seed := new File(".")

lazy val seeds = taskKey[Seq[File]]("Seed.")
seeds := List(new File("."))

sourceGenerators in Compile <+= seed

(sourceGenerators in Compile) <+= seed

sourceGenerators in Compile <+= Def.task(seed.value)

(sourceGenerators in Compile) <+= Def.task {
  seed.value
}

(sourceGenerators in Compile) <+= (Def.task {
  seed.value
})

sourceGenerators in Compile <++= seeds

(sourceGenerators in Compile) <++= seeds

sourceGenerators in Compile <++= Def.task(seeds.value)

(sourceGenerators in Compile) <++= Def.task {
  seeds.value
}

(sourceGenerators in Compile) <++= (Def.task {
  seeds.value
})
