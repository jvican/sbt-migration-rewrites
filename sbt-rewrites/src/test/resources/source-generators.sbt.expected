lazy val seeds = taskKey[Seq[File]]("Seed.")
seeds := List(new File("."))

sourceGenerators in Compile += seeds

(sourceGenerators in Compile) += seeds

sourceGenerators in Compile += Def.task(seeds.value)

(sourceGenerators in Compile) += (Def.task {
  seeds.value
})

(sourceGenerators in Compile) += (Def.task {
  seeds.value
})
